great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -449,6 +440,128 @@ class MetricDomainTypes.
                     "SqlAlchemyExecutionEngine only supports the great_expectations condition_parser."
                 )
 
+        if "column" in domain_kwargs:
+            return selectable
+
+        if "column_A" in domain_kwargs and "column_B" in domain_kwargs:
+            if "ignore_row_if" in domain_kwargs:
+                if self.active_batch_data.use_quoted_name:
+                    # If case matters...
+                    # noinspection PyPep8Naming
+                    column_A_name = quoted_name(domain_kwargs["column_A"])
+                    # noinspection PyPep8Naming
+                    column_B_name = quoted_name(domain_kwargs["column_B"])
+                else:
+                    # noinspection PyPep8Naming
+                    column_A_name = domain_kwargs["column_A"]
+                    # noinspection PyPep8Naming
+                    column_B_name = domain_kwargs["column_B"]
+
+                ignore_row_if = domain_kwargs["ignore_row_if"]
+                if ignore_row_if == "both_values_are_missing":
+                    selectable = (
+                        sa.select([sa.text("*")])
+                        .select_from(selectable)
+                        .where(
+                            sa.and_(
+                                sa.column(column_A_name) != None,
+                                sa.column(column_A_name) != None,
+                            )
+                        )
+                    )
+                elif ignore_row_if == "either_value_is_missing":
```

Comment:
Second test addition request: a test that shows proper sqlalchemy handling of *both* row_condition and ignore_row_if

Reply:
@jcampbell Please see the previous comments and the additional two test cases.  All of the "column_pair" and the "multicolumn" test cases utilize both, `row_condition` and `ignore_row_if` directives.